recipes/coin-utils/all/conanfile.py

```diff
@@ -1,10 +1,13 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile, tools
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.files import get, apply_conandata_patches, rm, rmdir, rename
+from conan.tools.scm import Version
+from conans import AutoToolsBuildEnvironment, tools
```

Comment:
```suggestion
from conans import AutoToolsBuildEnvironment
```
reimported

Reply:
Just import tools from conans as tools_legacy to not mask tools from conan.